app/api/v2/identity/sessions.rb

```diff
@@ -136,6 +136,87 @@ def get_user(email)
             error!({ errors: ['identity.session.auth0.invalid_params'] }, 422)
           end
         end
+
+        namespace :switch do
+          desc 'Switch user session',
+               success: { code: 200, message: 'Session was switched' },
+               failure: [
+                 { code: 404, message: 'Record is not found' }
+               ]
+          params do
+            optional :oid,
+                     type: String,
+                     desc: 'Organization OID'
+          end
+          post do
+            user = User.find_by(uid: session[:uid])
+            error!({ errors: ['identity.session.not_found'] }, 404) unless user
+
+            aid = params[:oid]
+            unless aid.nil?
+              # Check account in the organization that user belong to
+              members = Membership.joins('LEFT JOIN organizations ON organizations.id = memberships.organization_id')
```

Comment:
this logic deserves a specific helper with unit tests.

Reply:
For `unless aid.nil?` I do check that if :
- `params[:oid]` provided that mean user need to switch to org mode.
- `params[:oid]` not provide that mean user need to remove org mode and turn to be individual user.